source/detail/parser/x86_64.cpp

```diff
@@ -156,15 +156,16 @@ namespace smeagle::x86_64 {
     return result.str();
   }
 
+  // Unwrap and remove typedef
+  const auto remove_typedef = [](st::Type *t) {
```

Comment:
Let's convert this to a regular function `static st::Type* remove_typedef(st::Type *t)`

Reply:
All set.